src/components/widget/List/RawList.js

```diff
@@ -527,4 +584,4 @@ RawList.propTypes = {
   onCloseDropdown: PropTypes.func.isRequired,
 };
 
-export default onClickOutside(RawList);
```

Comment:
Removing this broke the lookup list dropdown hide/show functionality on clicking outside.

Reply:
I removed it because it didn't work as intended and implemented it myself. With `react-onclickoutside` you can't click on the scrollbar. I'll roll back the changes and post a screencast

Reply:
Basically this issue https://github.com/metasfresh/metasfresh-webui-frontend/issues/1605. Could be that your suggestion https://github.com/metasfresh/metasfresh-webui-frontend/pull/1652#discussion_r172499024 might fix that

Reply:
What about https://github.com/Pomax/react-onclickoutside#regulate-whether-or-not-to-listen-to-scrollbar-clicks ?

Reply:
That didn't seem to work, unfortunately